### ISSUE TITLE
Storing API keys in the client?

### ISSUE THREAD

**Issue by garbados** [chadtmiller/ember-orchestrate-adapter#6]
Hey, where does ember-orchestrate store your API key? In the readme, you set it like this...

``` javascript
// app/adapters/application.js
import OrchestrateAdapter from 'ember-orchestrate/adapter';

export default OrchestrateAdapter.extend({
  apiKey: 'Your api key'
});
```

...which leads me to believe it's stored in the client, but that's a major security vulnerability, so I want to make sure you're storing it elsewhere, securely.

**Comment by bigmoves** [comment]
@garbados thanks for pointing this out. I've addressed your concerns here #7. Since there really isn't a good way to securely store client-side tokens, I removed all support for it. The ember-orchestrate addon will rely on the included node-http-proxy to route requests to Orchestrate. Setting the API key will be no different than setting environment variables in a node application.

**Comment by garbados** [comment]
Cool! LGTM
